libcodechecker/analyze/analyzer.py

```diff
@@ -122,13 +147,26 @@ def perform_analysis(args, context, actions, metadata):
     LOG.info("Starting static analysis ...")
     start_time = time.time()
 
+    # Use Manager to create data objects which can be
+    # safely shared between processes.
+    manager = Manager()
+
+    cfg_mgr = manager.dict()
+    for k, v in config_map.items():
+        cfg_mgr[k] = v
+
+    config_map = cfg_mgr
```

Comment:
These few lines could be replaced by this:
```.py
config_map = manager.dict(config_map)
```
And why isn't `config_map` a `manager.dict` originally?

Reply:
Until it is not shared between multiple processes it is not necessary to create a proxy object.
I just wanted to create the proper types before they are shared between processes. This way it is more explicit, if it was created in another module it would be less visible.